packages/prosess-vilkar-sykdom/src/SykdomProsessIndex.tsx

```diff
@@ -0,0 +1,107 @@
+import avslattImage from '@fpsak-frontend/assets/images/avslaatt_hover.svg';
+import innvilgetImage from '@fpsak-frontend/assets/images/innvilget_hover.svg';
+import vilkarUtfallType from '@fpsak-frontend/kodeverk/src/vilkarUtfallType';
+import { FlexColumn, FlexContainer } from '@fpsak-frontend/shared-components/index';
+import FlexRow from '@fpsak-frontend/shared-components/src/flexGrid/FlexRow';
+import Image from '@fpsak-frontend/shared-components/src/Image';
+import VerticalSpacer from '@fpsak-frontend/shared-components/src/VerticalSpacer';
+import SideMenu from '@navikt/nap-side-menu';
+import classNames from 'classnames/bind';
+import { Element, EtikettLiten, Undertittel } from 'nav-frontend-typografi';
+import * as React from 'react';
+import { createIntl, createIntlCache, FormattedMessage, RawIntlProvider } from 'react-intl';
+import { dateFormat } from '@fpsak-frontend/utils';
+import messages from '../i18n/nb_NO.json';
+import styles from './sykdomProsessIndex.less';
+
+const cx = classNames.bind(styles);
+
+const cache = createIntlCache();
+
+const intl = createIntl(
+  {
+    locale: 'nb-NO',
+    messages,
+  },
+  cache,
+);
+
+const getVilkarOkMessage = originalErVilkarOk => {
+  let messageId = 'VilkarresultatMedOverstyringForm.IkkeBehandlet';
+  if (originalErVilkarOk) {
+    messageId = 'VilkarresultatMedOverstyringForm.ErOppfylt';
+  } else if (originalErVilkarOk === false) {
+    messageId = 'VilkarresultatMedOverstyringForm.ErIkkeOppfylt';
+  }
+
+  return (
+    <Element>
+      <FormattedMessage id={messageId} />
+    </Element>
+  );
+};
+
+const SykdomProsessIndex = ({ panelTittelKode, lovReferanse, vilkar }) => {
+  const [activeTab, setActiveTab] = React.useState(0);
+  const activeVilkår = vilkar[0];
```

Comment:
Litt usikker på om jeg skjønner denne... 

Hva er `vilkar`, hvorfor er den en liste og hvorfor henter vi alltid ut elementet fra index `0` for å finne periodene som har blitt vurdert? Å basere seg på hardkodet index er relativt error prone, så vil anbefale å unngå det hvis ikke vi må 😕 

Hvis vi må gjøre det sånn, så kan vi kanskje bare sende ned vilkårsobjektet til denne komponenten (verdien til `vilkar[0]`) istedetfor å sende ned hele lista (`vilkar`), så har vi hvertfall abstrahert det vekk herifra

Reply:
`vilkar` er en liste av alle vilkår som matcher `vilkarType.MEDISINSKEVILKÅR_UNDER_18_ÅR` som skal være bare ett, derfor index `0`. Kan flytte dette ut av komponenten